html_app/components/FACSModelFactory.js

```diff
@@ -101,10 +101,10 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
             function s_block(x) {
 				return Math.exp(-((2 - x) * Math.exp(2 - x) - .9) * ((2 - x) * Math.exp(2 - x) - .9) / .4);
             }
-//            function s_block_C(x){
-//                return Math.exp(-((1.3 - x) * Math.exp(1.3 - x) - .9) * ((1.3 - x) * Math.exp(1.3 - x) - .9)/ 0.4);
-//            }
             function s_block_C(x){
+                return Math.exp(-((0.8 - x) * Math.exp(1 - x) - .9) * ((0.8 - x) * Math.exp(1 - x) - .9)/ 0.4);
+            }
+            /*function s_block_C(x){
```

Comment:
Not a merge blocker, but going forward you should avoid committing commented code.